src/libutil/config.cc

```diff
@@ -432,10 +420,8 @@ OptionalPathSetting::OptionalPathSetting(Config * options,
 
 std::optional<Path> OptionalPathSetting::parse(const std::string & str) const
 {
-    if (str == "")
-        return std::nullopt;
-    else
-        return parsePath(*this, str);
+    if (str == "") return std::nullopt;
+    return parsePath(*this, str);
```

Comment:
I'm not convinced that this style is better. In any case it seems like unnecessary code churn.

Reply:
I dropped all if-return-else-return related changes in my PR.